src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx

```diff
@@ -2041,131 +2172,7 @@ If used, make sure to add the configuration flags just before calling `appToken`
 Here's a sample configuration:
 
 ```csharp
```

Comment:
@ndesai-newrelic Same here, can you provide a Xamarin sample config?

Reply:
public App ()
            {
                InitializeComponent();

                MainPage = new MainPage();
                Application.Current.PageAppearing += OnPageAppearing;
                Application.Current.PageDisappearing += PageDisappearing;

                CrossNewRelicClient.Current.HandleUncaughtException();
                CrossNewRelicClient.Current.TrackShellNavigatedEvents()
      // Set optional agent configuration
      // Options are: crashReportingEnabled, loggingEnabled, logLevel, collectorAddress, crashCollectorAddress
      AgentStartConfiguration agentConfig = new AgentStartConfiguration(true, true, LogLevel.INFO, "mobile-collector.newrelic.com", "mobile-crash.newrelic.com");
      if (Device.RuntimePlatform == Device.Android)
                {
                    //CrossNewRelicClient.Current.Start("<APP-TOKEN-HERE>");
                    // Start with optional agent configuration
                    CrossNewRelicClient.Current.Start("<APP-TOKEN-HERE", agentConfig);
                }
                else if (Device.RuntimePlatform == Device.iOS)
                {
                    //CrossNewRelicClient.Current.Start("<APP-TOKEN-HERE>");
                    // Start with optional agent configuration 
                    CrossNewRelicClient.Current.Start("<APP-TOKEN-HERE", agentConfig);
                }
            }
        }